docs/2.7.3.md

```diff
@@ -0,0 +1,31 @@
+# ProxySQL 2.7.3 and ProxySQL admin tools (2025-04-21)
+
+--8<--- "rn-block-top.md"
+
+## Release highlights
+
+In this release, we are skipping [ProxySQL 2.7.2]. We are releasing ProxySQL 2.7.3., which is the latest ProxySQL release. This decision ensures that you receive the most up-to-date features and improvements. Compared to the previous [ProxySQL 2.7.1-1 release], this release includes all improvements and bug fixes available in [ProxySQL 2.7.2] and [ProxySQL 2.7.3].
+
+* Added a configurable feature for logging and analyzing events and queries, providing real-time understanding of how applications are interacting with the database and its performance.
+
+* Added support for PMM metrics (`runtime_mysql_servers_*`) through the Prometheus exporter, helping you monitor and troubleshoot your database efficiently.
+
+* Introduced flexibility in query configurations with support for `@@session` and `@global` variables in `SET` statements.
+
+* Resolved collation mismatches between the client (frontend) and the server (backend) when collation IDs exceed 255, ensuring consistent query results.
+
+* Fixed issues with DELETE operations in the stats memory database, improving reliability during maintenance tasks.
+
+### Bug fixes
```

Comment:
```suggestion
```
What goes below is not a list of bugfixes, but a mere note. I suggest we remove the title.